x-pack/plugins/maps/public/classes/sources/es_source/es_source.js

```diff
@@ -163,16 +163,12 @@ export class AbstractESSource extends AbstractVectorSource {
       const esResp = await searchSource.fetch();
       esBounds = _.get(esResp, 'aggregations.fitToBounds.bounds');
```

Comment:
rather than keep hiding the root-cause, could we remove use of lodash here?

```
 esBonds = aggregations.fitToBounds.bounds ? aggregations.fitToBounds.bounds : null;
```

or 

```
if (!aggregations.fitToBounds.bounds) {
    return null;
} 
esBounds = aggregations.fitToBounds.bounds;
```

which helps clarity